packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterInputDate.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import { unstable_useId as useId } from '@mui/material/utils';
+import { GridLoadIcon } from '../../icons/index';
+import { GridFilterInputValueProps } from './GridFilterInputValueProps';
+
+export interface GridFilterInputDateProps extends GridFilterInputValueProps {
+  type?: 'date' | 'datetime-local';
+}
+
+export const SUBMIT_FILTER_STROKE_TIME = 500;
```

Comment:
I think we can re-use the variable `SUBMIT_FILTER_STROKE_TIME` of `GridFilterInputValue`
Or rename this one.
To avoid having two variables with the same name, one exported an not the other.

Reply:
I renamed it, because we are splitting components to create more independence between them